xbmc/interfaces/legacy/Player.h

```diff
@@ -493,6 +493,26 @@ namespace XBMCAddon
       void seekTime(double seekTime);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_Player
+      /// @brief \python_func{ addSubtitle(subtitleFile, name, language, activate) }
+      ///-----------------------------------------------------------------------
+      /// Add subtitle file with name and language and optionally activate it.
+      ///
+      /// @param subtitleFile        File to use as source ofsubtitles
+      /// @param name                Name of the subtitle to display
+      /// @param language            Language of the subtitlte to use
```

Comment:
```suggestion
      /// @param language            Language of the subtittle to use
```

Sorry, my bad, typo on my previous suggestion